HISTORY.rst

```diff
@@ -2,6 +2,43 @@
 History
 =======
 
+3.20.0 (2020-09-30)
+-------------------
+Tasks, Flows, and Automation:
+
+- ``cci flow list`` now displays flows in different groups that are organized by functional area. (This is similar to how ``cci task list`` currently works).
+- The ``insert_record`` task can now be used against the Tooling API. We clarified that this task can accept a dict of values if configured in ``cumulusci.yml``.
+- Added support for newer metadata components to the ``update_package_xml`` task.
+- We've deprecated the ``retrieve_scratch`` standard flow. ``retrieve_changes`` is the current recommended way for retrieving source-tracked changes.
```

Comment:
Say removed instead of deprecated. Deprecated sounds like it's still available.

Since this is a backwards-incompatible change, it should be called out under a "critical changes" heading at the beginning of the release notes